package.json

```diff
@@ -167,6 +168,7 @@
     "event-stream": "3.3.2",
     "expect.js": "0.3.1",
     "faker": "1.1.0",
+    "fs-extra": "0.30.0",
```

Comment:
No longer needed as need for backup of bundle file is removed as now using reference to language files.